apps/backend/src/websocket/server.py

```
@@ -0,0 +1,246 @@
+"""
+
+import asyncio
+import logging
+import json
+from typing import Any, Dict
+from fastapi import APIRouter, WebSocket, WebSocketDisconnect, HTTPException
+
+from .connection_manager import get_connection_manager, ConnectionManager
+from .message_protocol import (
+    MessageType,
+    WebSocketMessage,
```

Ruff (0.12.2)

`.message_protocol.WebSocketMessage` imported but unused

Remove unused import

(F401)